### PR TITLE
Default page layout

### LINKED ISSUE #1706: [Global] Styling for default pages within the site.
They be brokenz.

Need to address styling for all pages throughout the site that don't have a specific template and Drupal defaults to chucking the content into the $content vars in the page.tpl.php.

### LINKED ISSUE #1949: Content sits behind navigation bar
For items like the campaign unsignup form (which is only available to staff), it becomes unusable.

screen shot 2014-05-02 at 11 46 06 am

### LINKED ISSUE #2164: Style the Beta access denied page
Go to https://www.dosomething.org/node/2012/confirmation as an anonymous user and it looks nasty. Maybe display a picture or video of Puppet Sloth?

screen shot 2014-05-16 at 10 37 44 am

### LINKED ISSUE #2197: /User page styling
Please update form field styling (width) on dosomething.org/user. 

screen shot 2014-05-19 at 2 08 10 pm

### LINKED ISSUE #2198: Reset password: Styling
This is the page users see AFTER they have clicked on the email link to reset their password. Yipes. Should be styled as purple messaging bar or basic text page. 

screen shot 2014-05-19 at 2 09 50 pm

### PR DESCRIPTION
**Pull request by DFurnes**
### Changes
- Sets a default page layout (with purple header & appropriate content margins), and adds theme suggestion to allow overriding page template by node type.
- Partializes "header" pattern to dry up that markup.
- Applies style classes directly to navigation in preprocess step, rather than styling based on `.node-type` page classes.

Fixes #1706. Also fixes #2198, fixes #2197, fixes #2164, and fixes #1949.

### CODE REVIEW

**Comment by DFurnes** [comment]
Example of a "default" page layout:

screen shot 2014-06-11 at 10 16 53 am

File: lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc [hunk 1]
```diff
@@ -66,6 +87,15 @@ function paraneue_dosomething_preprocess_page(&$vars) {
       'links' => $footer_links
     )
   );
+
+
+  // Misc
```
Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]:
```
/** 
 * Drupal Comments!
 */
```

**Comment by aaronschachter** [comment]
Looks ok by me :+1:

**Comment by barryclark** [comment]
Looks good @DFurnes :sparkles: :sparkles: 

Would be good to do a search for all relevant issues that you can close with this PR. Combo time.

**Comment by angaither** [comment]
MULTIBALL :baseball: :basketball: :football: :sunglasses: 

cc: @blisteringherb

**Comment by DFurnes** [comment]
:swimmer: :walking: :speech_balloon: :bangbang:

**Comment by weerd** [comment]
Late to the game! :+1: Looking forward to this merging :)

**Pull request merged**